pl_bolts/callbacks/ssl_online.py

```diff
@@ -102,16 +102,16 @@ def on_train_batch_end(
         representations = representations.detach()
 
         # forward pass
-        mlp_preds = pl_module.non_linear_evaluator(representations)  # type: ignore[operator]
-        mlp_loss = F.cross_entropy(mlp_preds, y)
+        mlp_logits = pl_module.non_linear_evaluator(representations)  # type: ignore[operator]
```

Comment:
If we look at the [SSLEvaluator](https://github.com/PyTorchLightning/lightning-bolts/blob/2ba3f62709b50604c29448f4ad620449796bce06/pl_bolts/models/self_supervised/evaluator.py) we find that it does not return logits.

The name should reflect what the output is :)